src/language-handlebars/clean.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+module.exports = function(ast, newNode) {
+  delete newNode.loc;
+  delete newNode.selfClosing;
+
+  // (Glimmer/HTML) ignore TextNode whitespace
+  if (ast.type === "TextNode") {
+    if (ast.chars.replace(/\s+/, "") === "") {
```

Comment:
how about trim()?

Reply:
Just for you information - trim !== .replace(/\s+/, ""), trim === `/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g, but here we can use it

Reply:
good to know :)
I'll give it a try.

Reply:
my middle-term goal it to remove that to make it whitespace sensible (like html)